basic/src/test/java/org/wildfly/ejbclient/testsuite/integration/basic/security/WhoAmITestCase.java

```diff
@@ -102,4 +104,20 @@ public void isCallerInRole2() throws Exception {
             Assert.assertFalse("amIInRole should return false for roles which don't exist", whoami.amIInRole("Dunstabzugshaube"));
         }
     }
+
+    /**
+     * Regression tests for WEJBHTTP-119
+     *
+     * EJB prefix is used in case of WILDFLY_NAMING_CLIENT
+     */
+    @Test
+    public void isCallerInRoleNameWithEjbPrefix() throws Exception {
+        Assume.assumeTrue(TestEnvironment.getAuthenticationType().equals(ServerAuthenticationType.USER));
+        Assume.assumeTrue(TestEnvironment.getContextType() == EJBClientContextType.WILDFLY_NAMING_CLIENT); // other context types uses ejb prefix by default
```

Comment:
The point here is that InitialContextDirectoryWildFlyNamingClient doesn't contains `ejb:` prefix (see [here](https://github.com/wildfly/ejb-client-testsuite/blob/main/basic/src/test/java/org/wildfly/ejbclient/testsuite/integration/basic/utils/jndi/InitialContextDirectoryWildFlyNamingClient.java#L70)), but other InitialContextDirectories does (like [here](https://github.com/wildfly/ejb-client-testsuite/blob/main/basic/src/test/java/org/wildfly/ejbclient/testsuite/integration/basic/utils/jndi/InitialContextDirectoryGlobal.java#L135)).

Reply:
I think we need a test with JNDI lookup since the lack of translation of ":local" address discovered on the server during the lookup to actual server address was the cause of the failure